File: Core/HouseKeepingAccounting.DAL/Models/House.cs
﻿using System.ComponentModel.DataAnnotations;

namespace HouseKeepingAccounting.DAL.Models
{
    public class House
    {
        [Key]
        public int Id { get; set; }
        public string CityName { get; set; }
        public string StreetName { get; set; }

        public string HomeNumber { get; set; }

        public Counter Counter { get; set; }
    }
}

File: Presentation/Frontend/house-keeping-account-react/src/Components/House/House.js
import React from 'react'
import './House.css'

const house = (props) =>{
    return(
        <div className="House">
            <p>City name: {props.cityName}</p>    
            <p>Street name: {props.streetName}</p>
            <p>Home number: {props.homeNumber}</p>         
            <p>{props.children}</p>   
            <input type="text" onChange={props.changed} value={props.cityName}/>
        </div>
    )
};

export default house;